mlrun/runtimes/databricks_job/databricks_runtime.py

```diff
@@ -137,3 +160,25 @@ def run(
 handler_arguments = json.loads(handler_arguments)
 
 """
+
+artifacts_code_template = """\n
+def mlrun_log_artifact(name, path):
```

Comment:
This entire code is not really protective in any way - you don't verify that the `path` or the `name` is a string, and you don't really check whether any of the os calls (reading, creating, writing files) actually succeed or not. The end result here may be that user's code would explode due to a problem in our code (which the user may or may not understand). 
In general, this entire code should be treated as a best-effort and it should at least be wrapped with a `try/except` to catch exceptions thrown in between. Should also consider more specific checks about things you can think of.